docs/GettingStarted/configuration/PelorusCore.md

```diff
@@ -0,0 +1,222 @@
+```
+
+## List of all configuration options
+
+| Variable | Required | Default Value |
+|----------|----------|---------------|
+| [[exporters]](PelorusExporters.md) | yes | - |
+| [prometheus_retention_size](#prometheus_retention_size) | no | `1GB` |
+| [prometheus_retention](#prometheus_retention) | no | `1y` |
+| [prometheus_storage](#promtheus-storage) | no | `false` |
+| [prometheus_storage_pvc_capacity](#prometheus_storage_pvc_capacity) | no | `2Gi` |
+| [prometheus_storage_pvc_storageclass](#prometheus_storage_pvc_storageclass) | no | `gp2` |
+| [openshift_prometheus_htpasswd_auth](#openshift_prometheus_htpasswd_auth)  | no | `internal:{SHA}+pvrmeQCmtWmYVOZ57uuITVghrM=` |
+| [openshift_prometheus_basic_auth_pass](#openshift_prometheus_basic_auth_pass) | no | `changeme` |
+| [[extra_prometheus_hosts]](#extra_prometheus_hosts) | no | - |
+| [thanos_version](#thanos_version) | no | `v0.28.0` |
+| [bucket_access_point](#bucket_access_point) | no | - |
+| [bucket_access_key](#bucket_access_key) | no | - |
+| [bucket_secret_access_key](#bucket_secret_access_key) | no | - |
+| [thanos_bucket_name](#thanos_bucket_name) | no | `thanos` |
+| [custom_ca](#custom-pki) | no  | - |
+
+## Prometheus
+
+Pelorus allows to configure few aspects of [Prometheus](https://prometheus.io/), that is deployed as an [Prometheus Operator](https://github.com/prometheus-operator/prometheus-operator) available from the [OLM dependency](https://olm.operatorframework.io/docs/concepts/olm-architecture/dependency-resolution/) mechanism.
+
+For detailed information about planning Prometheus storage capacity and configuration options please refer to the [operational aspects](https://prometheus.io/docs/prometheus/latest/storage/#operational-aspects) of the Prometheus documentation.
+
+### Prometheus Data Retention
+
+###### prometheus_retention_size
+
+- **Required:** no
+- **Default Value:** 1GB
+
+: Users have the option to configure maximum size of storage to be used by Prometheus. The oldest data will be removed first if it exceeds that limit. Even if the data is within retention time, but over retention size, it will also be removed.
+
+: Units supported: MB, GB, TB, PB, EB
+
+###### prometheus_retention
+
+- **Required:** no
+- **Default Value:** 1y
+
+: Prometheus is removing data older than 1 year, so if the metric you are interested in happened to be older than 1 year it won't be visible.
+
+: Units supported: d, y
+
+
+### Prometheus Persistent Volume
+
+Unlike ephemeral volume that have a lifetime of a pod, persistent volume allows to withstand container restarts or crashes making Prometheus data resilient to such situations. Pelorus allows to use underlying [Prometheus Operator Storage](https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/storage.md#storage-provisioning-on-aws) capabilities by using Kubernetes [`StorageClass`](https://kubernetes.io/docs/concepts/storage/storage-classes/).
```

Comment:
change link https://prometheus-operator.dev/docs/operator/storage/#storage-provisioning-on-aws

Reply:
No, because this one is on AWS and we do want to provide instruction for any S3 compatible service.

Reply:
it is the same content, but instead of pure markdown, it is in their docs

Reply:
ok